schemas/value_sources.json

```
@@ -173,5 +179,26 @@
         "$ref": "https://eq.ons.gov.uk/value_sources.json#/calculated_summary_value_source"
       }
     ]
+  },
+  "progress_value_source": {
+    "description": "A reference to a progress object property, for a section or a block. When resolved it returns the string COMPLETED or null.",
```

Would be clearer to Author as:
```suggestion
    "description": "The progression status of a section or a block. When resolved it returns the string COMPLETED or null.",
```

Will it return null?. It should be any status or Null?

---

FIxed